Sketch.java

```diff
@@ -1,36 +1,191 @@
 import processing.core.PApplet;
 
+/**
+ * Arrays
+ * @author D. Ursu
+ */
+
 public class Sketch extends PApplet {
 	
-	
+	float[] circleY = new float[10];
+  float [] circleX =  new float [10];
+  float playerY = 350;
+  float playerX = 200;
+  boolean [] ballHideStatus = new boolean [10];
+  int lives = 3;
+  boolean downPressed = false;
+  boolean upPressed = false;
+  boolean wPressed = false;
+  boolean aPressed = false;
+  boolean sPressed = false;
+  boolean dPressed = false;
+
   /**
    * Called once at the beginning of execution, put your size all in this method
    */
   public void settings() {
 	// put your size call here
     size(400, 400);
+
+    for (int x = 0; x < circleX.length; x++){
+      circleX[x] = random(width);
+      circleY[x] = random(height);
+    }
   }
 
   /** 
    * Called once at the beginning of execution.  Add initial set up
    * values here i.e background, stroke, fill etc.
    */
   public void setup() {
-    background(210, 255, 173);
+    for (int i = 0; i < circleY.length; i++) {
+      ballHideStatus[i] = false;
+    }
   }
 
   /**
    * Called repeatedly, anything drawn to the screen goes here
    */
   public void draw() {
-	  
-	// sample code, delete this stuff
-    stroke(128);
-    line(150, 25, 270, 350);  
+    background(0);
+
+    for (int i = 0; i < circleY.length; i++) {
+      if (ballHideStatus[i] == false){
+        fill(255);
+        ellipse(circleX[i], circleY[i], 30, 30);
+        stroke(0);
+  
+        circleY[i] += 2;
+ 
+      if(circleY[i] > height) {
+        circleY[i] = 0;
+      }
+      else if(ballHideStatus[i] == true){
+        fill(0);
+        ellipse(circleX[i], circleY[i], 25, 25);
+        stroke(0);
+      }
+    
+      if((playerX <= circleX[i] + 20) && (playerX >= circleX[i] - 20)){
+        if((playerY <= circleY[i] + 20) && (playerY >= circleY[i] - 20)){
```

Comment:
based on this logic, you can still collide with a circle that is no longer visible